targets/TARGET_NORDIC/TARGET_NRF5x/app_uart.h

```diff
@@ -0,0 +1,98 @@
+/* mbed Microcontroller Library
```

Comment:
This seems to be a copy of the app_uart module from the Nordic SDK

Reply:
Not quite. Much of it was cherry picked from the original  Nordic SDK file to satisfy minimum requirements of the SDK logging interface, the rest is re-implemented to suit the redirection to stdin/stdout.

Reply:
Sure, Nordic's license header should be kept though